WalletWasabi.Tests/UnitTests/ViewModels/UiContextBuilder.cs

```diff
@@ -6,6 +6,7 @@
 using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
 using WalletWasabi.Fluent.ViewModels.Navigation;
 using WalletWasabi.Tests.UnitTests.ViewModels.TestDoubles;
+using WalletWasabi.Tests.UnitTests.ViewModels.UIContext;
```

Comment:
Same here. In my IDE this is just an unnecessary `using`

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/11447#discussion_r1320149525